src/snowflake/connector/pandas_tools.py

```diff
@@ -178,6 +183,50 @@ def write_pandas(
     else:
         columns = ",".join(list(df.columns))
 
+    if auto_create_table:
+        file_format_name = None
+        while True:
```

Comment:
why is a loop needed? in case `file_format_name` already exists?

Reply:
Yea in case it already exists. We have a similar loop for the stage